### PR TITLE
Added Pakistan Flag

### PR DESCRIPTION
**Pull request by ObaidAshraf**
Flag of Pakistan

### CODE REVIEW

**Comment by ManrajGrover** [comment]
@ObaidAshraf Hey! This looks great! But it differs from one mentioned in [Wikipedia](https://en.wikipedia.org/wiki/Flag_of_Pakistan). Can you share any source where it mentions to be official?

**Comment by ObaidAshraf** [comment]
@ManrajGrover .. Hey! Thankyou for replying..
I have modified the Flag of Pakistan. Now, it looks pretty much same as that on Wikipedia link..

File: Flags/pakistan.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
```
Comment (ManrajGrover) [inline Flags/pakistan.html]: @ObaidAshraf Hey! Please indent the html file properly along with CSS.

**Comment by ManrajGrover** [comment]
@ObaidAshraf Sorry for late reply. I have left 1 comment. Please fix that so that we can merge. :)

**Comment by ObaidAshraf** [comment]
@ManrajGrover I have modified the indentation in HTML file. Sorry for too many errors !!

**Comment by ManrajGrover** [comment]
@ObaidAshraf No issues. :smile: You forgot to indent the CSS. It should be a quick fix. Will merge as soon as you indent it. :)

**Comment by ObaidAshraf** [comment]
@ManrajGrover I've added indentation of CSS .. I hope it's complete now :D ..

**Comment by ManrajGrover** [comment]
@ObaidAshraf Thank you for contributing! :smile: Merged this.

**Pull request merged**